frontend_tests/node_tests/muting.js

```diff
@@ -73,14 +73,14 @@ run_test("get_and_set_muted_topics", () => {
     assert.deepEqual(muting.get_muted_topics().sort(), [
```

Comment:
It doesn't make sense that these changes in tests are required for changes I did in this PR. @andersk FYI